wagtail/wagtailadmin/views/home.py

```diff
@@ -10,6 +12,11 @@
 from wagtail.wagtailcore.models import Page, PageRevision, UserPagePermissionsProxy
 
 
+User = get_user_model()
+pk_field_name = User._meta.pk.name
+pk_field = User._meta.get_field(pk_field_name)
```

Comment:
I used `get_field()` because that is the documented api.  I can use `User._meta.pk` directly if you prefer. I'll push the update once the merge tests are successful